engine/access/rpc/backend/backend_scripts.go

```diff
@@ -90,18 +96,28 @@ func (b *backendScripts) executeScriptOnExecutionNode(
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "failed to find execution nodes at blockId %v: %v", blockID.String(), err)
 	}
+	// encode to MD5 as low compute/memory lookup key
+	encodedScript := md5.Sum(script) //nolint:gosec
 
 	// try each of the execution nodes found
 	var errors *multierror.Error
 	// try to execute the script on one of the execution nodes
 	for _, execNode := range execNodes {
+		executionTime := time.Now()
```

Comment:
actually, this too can be inside the `if b.log.GetLevel` block since we don't really care about the exact time when the script was executed.